src/pages/index.tsx

```diff
@@ -1,3 +1,62 @@
+import { Post } from "@/shared/components/index";
+import { useEffect, useState } from "react";
+import styles from "@/pages/index.module.scss";
+import instance from "./api/axiosInstance";
+
+type Res = {
+  item: {
+    id: string;
+    hourlyPay: number;
+    startsAt: string;
+    workhour: number;
+    description: string;
+    closed: boolean;
+    shop: {
+      item: {
+        id: string;
+        name: string;
+        category: string;
+        address1: string;
+        address2: string;
+        description: string;
+        imageUrl: string;
+        originalHourlyPay: number;
+      };
+    };
+  };
+};
+
+type Props = {
+  item: Res[];
+};
+
 export default function Home() {
-  return <div>메인페이지</div>;
+  const [userData, setUserData] = useState<Props["item"]>([]);
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await instance.get(`notices`);
+      setUserData(response.data.items);
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      {userData.map((data: Res) => (
+        <div className={styles.a} key={data.item.id}>
+          <Post
+            imageUrl={data.item.shop?.item?.imageUrl}
+            key={data.item.id}
+            startsAt={data.item.startsAt}
+            workhour={data.item.workhour}
+            hourlyPay={data.item.hourlyPay}
+            closed={data.item.closed}
+            name={data.item.shop?.item?.name}
+            address1={data.item.shop?.item?.address1}
+            originalHourlyPay={data.item.shop?.item?.originalHourlyPay}
+          />
+        </div>
```

Comment:
구조분해할당도 적극적으로 활용해주시면 좋을 것 같습니다!
```suggestion
      {userData.map((data: Res) => {
        <div className={styles.a} key={data.item.id}>
          const {id, startsAt, workhour, hourlyPay, closed} = data.item;
          
        return (
          <Post
            imageUrl={shop?.item?.imageUrl}
            key=id}
            startsAt={startsAt}
            workhour={workhour}
            hourlyPay={hourlyPay}
            closed={closed}
            name={shop?.item?.name}
            address1={shop?.item?.address1}
            originalHourlyPay={shop?.item?.originalHourlyPay}
          />
          )
        </div>
        }
```